error: use of undeclared identifier 'QStandardPaths'

<!--- Provide a general summary of the issue in the title above -->

## Expected Behavior
Should be built successfully

## Current Behavior
The following error occurs when build process is around 34%

[ 34%] Building CXX object src/CMakeFiles/keepassx_core.dir/core/EntryAttachments.cpp.o
/Users/william/Documents/project/keepassxc/src/core/Config.cpp:98:47: error: use of undeclared identifier 'QStandardPaths'
        userPath = QDir::fromNativeSeparators(QStandardPaths::writableLocation(QStandardPaths::DataLocation));
                                              ^
/Users/william/Documents/project/keepassxc/src/core/Config.cpp:98:80: error: use of undeclared identifier 'QStandardPaths'
        userPath = QDir::fromNativeSeparators(QStandardPaths::writableLocation(QStandardPaths::DataLocation));
                                                                               ^
2 errors generated.
make[2]: *** [src/CMakeFiles/keepassx_core.dir/core/Config.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/CMakeFiles/keepassx_core.dir/all] Error 2
make: *** [all] Error 2
 
## Steps to Reproduce (for bugs)
1. Remove everything in **build** directory
2. Use the following cmake
cmake -DCMAKE_OSX_ARCHITECTURES=x86_64 -DCMAKE_BUILD_TYPE=Release -DWITH_XC_AUTOTYPE=ON -DWITH_XC_NETWORKING=ON -DWITH_XC_BROWSER=ON -DWITH_XC_SSHAGENT=ON -DCMAKE_PREFIX_PATH=/usr/local/Cellar/qt5/5.10.0_1/lib/cmake ..
3. make -j8 package
4. The above error occurs

## Debug Info
Latest source code from this repository.

Operating system: Mac OS 10.13.3
CPU architecture & Kernel: Darwin MacbookPro 17.3.0 Darwin Kernel Version 17.3.0: Thu Nov 9 18:09:22 PST 2017; root:xnu-4570.31.3~1/RELEASE_X86_64 x86_64

--- a/src/core/Config.cpp
+++ b/src/core/Config.cpp
@@ -22,6 +22,7 @@
 #include <QDir>
 #include <QSettings>
 #include <QTemporaryFile>
+#include <QStandardPaths>
 
 Config* Config::m_instance(nullptr);
 

